proof/ec_mult.saw

```diff
@@ -396,7 +392,7 @@ invariant_booth'_thm <-
   custom_prove_cryptol
   {{ \p1 booth_index -> POINTonE1_invariant (booth' (precomputed_table p1) booth_index) == True }}
   do { unfolding ["booth'", "booth_lookup"];
-       simplify (addsimp (nth POINTonE1_thms 1) (cryptol_ss ())); // the "invariant_rep" theorem
+       simplify (addsimp (nth POINTonE1_thms 2) (cryptol_ss ())); // the "invariant_rep" theorem
```

Comment:
This looks very brittle.  Can we just simplify with the whole set of POINTonE1thms?  Or can we define the invariant_rep theorem just after the list is defined, so it will be less easy to forget to change?

Reply:
looks like it works with the whole set too

Reply:
hum, actually no it doesn't...